components/experimental/src/transliterate/provider.rs

```diff
@@ -65,7 +66,7 @@ impl<'de> serde::Deserialize<'de> for RuleBasedTransliterator<'de> {
             #[serde(borrow)]
             pub id_group_list: VarZeroVec<'a, VarZeroSlice<SimpleIdULE>>,
             #[serde(borrow)]
-            pub rule_group_list: VarZeroVec<'a, VarZeroSlice<RuleULE>>,
+            pub rule_group_list: VarZeroVec<'a, VarZeroSlice<RuleULE, Index32>, Index32>,
```

Comment:
Nit: probably only 1 of these needs to be Index32.

It looks like the biggest are Hant to Hans, Hans to Hant, and Hani to Latn.